src/Symfony/Component/Validator/Tests/Constraints/GreaterThanOrEqualValidatorWithPositiveOrZeroConstraintTest.php

```diff
@@ -98,10 +99,10 @@ public function testValidComparisonToPropertyPath($comparedValue)
     }
 
     /**
-     * @dataProvider provideValidComparisonsToPropertyPath
+     * @dataProvider throwsOnInvalidStringDatesProvider
      */
-    public function testValidComparisonToPropertyPathOnArray($comparedValue)
+    public function testThrowsOnInvalidStringDates(AbstractComparison $constraint, $expectedMessage, $value)
```

Comment:
I didn't change the method. I removed an useless skipped test (from a method that doesn't exist anymore) and added the new one.